sentry-android-core/src/main/java/io/sentry/android/core/adapters/DateSerializerAdapter.java

```diff
@@ -8,7 +8,7 @@
 import java.lang.reflect.Type;
 import java.util.Date;
 
-class DateSerializerAdapter implements JsonSerializer<Date> {
+public class DateSerializerAdapter implements JsonSerializer<Date> {
```

Comment:
Adapters need to be public? Is it so that we can move to a different package for organization?

Reply:
yes, just moved for organization, the number of adapters is growing, let's deal with the public access later on